src/Microsoft.ServiceFabric.Actors/FabricActorExceptionKnownTypes.cs

```diff
@@ -83,30 +82,6 @@ internal class FabricActorExceptionKnownTypes
                         InnerExFunc = ex => GetInnerExceptions(ex),
                     }
                 },
-                {
-                    "Microsoft.ServiceFabric.Actors.Migration.Exceptions.ActorCallsDisallowedException", new ConvertorFuncs()
-                    {
-                        ToServiceExFunc = ex => ToServiceException(ex),
-                        FromServiceExFunc = (svcEx, innerEx) => FromServiceException<ActorCallsDisallowedException>(svcEx, innerEx),
```

Comment:
If caller service is on the new version and don't know these exceptions and callee service is on the old version and throws some of these, could we have a problem?